mlir-tensorrt/executor/lib/Support/DeviceInfo.cpp

```diff
@@ -23,26 +23,33 @@
 
 using namespace mlirtrt;
 
+namespace {
```

Comment:
nit: We use the LLVM convention (https://llvm.org/docs/CodingStandards.html#anonymous-namespaces), which is to declare functions `static` rather than wrapping in anonymous namespace. Only local classes/structs would get wrapped in anonymous namespace.

Reply:
Oh I see, wasn't aware. Fixed now.